darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -297,7 +301,7 @@ def __init__(
             pass
 
         # TODO: remove below in the next version ======>
-        accelerator, gpus, auto_select_gpus = self._extract_torch_devices(
+        accelerator, devices, auto_select_gpus = self._extract_torch_devices(
```

Comment:
Could we actually remove support for `torch_device_str`, and also remove these lines (as suggested in the TODO comment)?